content/en/blog/2025/otel-js-sdk-2-0.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Announcing the OpenTelemetry JavaScript SDK 2.0
+linkTitle: OTel JS SDK 2.0
+date: 2025-02-21
```

Comment:
Setting draft status so that we can see the preview of this post (might have to change the date too, I'm not sure):

```suggestion
date: 2025-02-21
draft: true
```